client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Bluehost from './bluehost';
+import SiteGround from './siteground';
+import SectionHeader from 'components/section-header';
+
+class HostCredentialsPage extends Component {
+	static propTypes = {
+		hostSlug: PropTypes.string.isRequired,
+	};
+
+	state = { fieldValues: {} };
```

Comment:
In fields.jsx you'd end up with:
```js
<FormTextInput
    id="email"
    ref={ props.fieldRef }
    value={ props.value }
    onChange={ props.onChange }
    placeholder={ props.translate( 'Email address' ) } />
```

`fieldRef` because you probably cannot get the `ref` prop from the parent component and if you did it might lead to reader confusion.